test/api/units_test.rb

```diff
@@ -124,33 +124,6 @@ def assert_tutorial_model_response(response, expected)
   end
 
 
-  def test_addtutorial_to_unit
-    count_tutorials = Tutorial.all.length
```

Comment:
Why are we not testing how tutorials are added to units? This is pretty important functionality.